frontend/components/Form.js

```diff
@@ -257,6 +283,14 @@ class Form extends Component {
               <span className="file-label">Choose a file...</span>
             </span>
           </label>
+          {this.state.hasImg ? (
+            <span>
+              {' '}
+              <Icon name="check-circle-green" size="1.8rem" alt="checkbox"></Icon>
+              {'   '}
```

Comment:
More than one space won't make a difference
```suggestion
              {' '}
```